tests/rules/avoid-cyclic-effects.test.ts

```diff
@@ -118,7 +118,85 @@ ruleTester().run(path.parse(__filename).name, rule, {
           private actions$: Actions,
         ) {}
       }
-    `),
+    `,
+    // https://github.com/timdeschryver/eslint-plugin-ngrx/issues/223
+    `
+      import { Injectable } from '@angular/core';
```

Comment:
Could we make the setup here simpler, or do you think it's worth to keep the full example?
I think it's easier to read if we can get rid of the service and some other boilerplate. something like this:

```ts
enum OrderEntityActionTypes {
        postPortInData = '[Order Entity] Post PortIn Data',
        postPortInDataSuccess = '[Order Entity] Post PortIn Data Success',
      }
      class PostPortInData implements Action {
        public readonly type = OrderEntityActionTypes.postPortInData
      }
      class PostPortInDataSuccess implements Action {
        public readonly type = OrderEntityActionTypes.postPortInDataSuccess
      }

        public postPortInData$ = createEffect(() =>
          this.actions$.pipe(
            ofType<PostPortInData>(OrderEntityActionTypes.postPortInData),
            switchMap(() => of(new PostPortInDataSuccess())),
          ),
        )
```